### ISSUE TITLE
Failed to install the IBM helm charts

### ISSUE THREAD

**Issue by rohitkrai03** [redhat-developer/redhat-helm-charts#14]
# Steps to Reproduce:
1. Go to +Add -> Dev Catalog
2. Check the Helm Charts checkbox

## For 4.5 cluster
3. Try to install the IBM helm charts but you will observe errors as follows:
a. For all the charts:

```
Failed to install helm chart: chart requires kubeVersion: >=1.10.1 which is incompatible with Kubernetes v1.18.0-rc.1
```

## For 4.4 cluster
3. Try to install the IBM helm charts but you will observe errors as follows:
a. Ibm Object Storage Plugin chart gets installed successfully,
b. For `Ibm Oms Ent Prod` and `Ibm Oms Pro Prod` charts we get below error

```
Failed to install helm chart: template: ibm-oms-ent-prod/templates/NOTES.txt:44:17: executing "ibm-oms-ent-prod/templates/NOTES.txt" at <(hasKey $imgConf "purpose") and (eq $imgConf.purpose "xapirest")>: can't give argument to non-function hasKey $imgConf "purpose"
```

### Actual Results:
We are unable to install the IBM charts.

### Expected Results:
IBM charts should get installed.

**Comment by rohitkrai03** [comment]
cc: @pedjak @sbose78

**Comment by pedjak** [comment]
@morstad can you take a look?

**Comment by morstad** [comment]
Believe the 4.5 issue is due to kubernetes release being a pre-release  :
- https://github.com/helm/helm/issues/3810#issuecomment-379877753
- https://github.com/Masterminds/semver/blob/0c3415da09dfa43a604d0945fc94c449e49438b5/README.md#working-with-pre-release-versions

**Comment by morstad** [comment]
@rohitkrai03 Not able to recreate issue reported for oms - please provide parameters specified.

**Comment by rohitkrai03** [comment]
I am just using the default values with licence as `true`.

**Comment by morstad** [comment]
@rohitkrai03 If I specify only license as ```true``` I see 'No object visited' as no resources to install.   If I specify 
```
  license: true
  license_store_call_center: true
```
then proceeds - please try setting both and I will spin up newer OCP nightly build as not seeing same as you.

**Comment by rohitkrai03** [comment]
@morstad I just tried it out with `4.4.0-0.nightly-2020-04-04-025830` cluster and got the same error i got earlier with `4.4-rc11`. Check the below screenshots - 

![Screenshot from 2020-04-30 17-57-17](https://user-images.githubusercontent.com/6041994/80710456-66df4c80-8b0c-11ea-9e99-488ea266e142.png)

![Peek 2020-04-30 17-58](https://user-images.githubusercontent.com/6041994/80710487-752d6880-8b0c-11ea-9efa-a9d27f256608.gif)

**Comment by morstad** [comment]
@rohitkrai03 The install is showing README in your screenshot - this is 4.5?  In 4.4 only get description from Chart.yaml on install.
![image](https://user-images.githubusercontent.com/24810680/80711441-f9083600-8ab5-11ea-9b04-90ef965b37e0.png)

**Comment by rohitkrai03** [comment]
Yeah, that's actually my local setup running on a 4.4 cluster. So the latest UI that is going to be there in 4.5 is visible. Sorry for the confusion.

**Comment by pedjak** [comment]
is this fixed meanwhile? can we close the issue?
